### ISSUE TITLE
New post success was a false positive 

### ISSUE THREAD

**Issue by lewri** [K1rakishou/Kuroba-Experimental#1038]
I went to make a new post. However, upon posting it directed me to a different thread and my post didn't actually succeed, which was quite bizarre.

The contents of my post got discarded (due to the false positive) but yeah, thought you'd like to know.

[logs.txt](https://github.com/K1rakishou/Kuroba-Experimental/files/14623094/logs.txt)

**Comment by K1rakishou** [comment]
You tried to create a new thread, right?

**Comment by lewri** [comment]
> You tried to create a new thread, right?

Yes I did. It targeted a thread made 30 minutes earlier and thought that was it (bookmarked it too and marked as my post)

**Comment by azumukupoe** [comment]
Yeah this is quite annoying
